src/components/ChooseMint.vue

```diff
@@ -33,16 +33,13 @@
 <script>
 import { defineComponent, ref } from "vue";
 import { getShortUrl } from "src/js/wallet-helpers";
+import {mapActions, mapState} from "pinia";
+import { useMintsStore } from "stores/mints";
 export default defineComponent({
   name: "ChooseMint",
   mixins: [windowMixin],
   props: {
-    proofs: Array,
-    activeProofs: Array,
-    mints: Array,
     tickerShort: String,
-    activeMintUrl: String,
-    activateMint: Function,
   },
   data: function () {
     return {
```

Comment:
Need to move `chosenMint` to computed state, otherwise the invoices flow is broken

Reply:
Fixed.